.github/workflows/velox_weekly.yml

```diff
@@ -206,3 +206,22 @@ jobs:
           echo "JAVA_HOME: $JAVA_HOME"
           cd $GITHUB_WORKSPACE/tools/gluten-it
           $TPCH_TEST
+
+  build-on-macos:
+    strategy:
+      fail-fast: false
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: macos-latest
```

Comment:
Can we target to a specific runner version?

https://docs.github.com/en/actions/reference/runners/github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
missed this. will cover this in the followup